.gitignore

```diff
@@ -1,6 +1,6 @@
 .*
 !.gitignore
-gradle/
+!gradle/
 gnd/debug/
 gnd/release/
 **/google_maps_api.xml
```

Comment:
How is gnd/src/mock/res/values/google_maps_api.xml included if this rule is present? Should we create an ignore override to include it?

Reply:
I did `git add -f` to force add `google-services.json` and `google_maps_api.xml`. I'll update the .gitignore too.